src/site/content/en/blog/file-system-access/index.md

```diff
@@ -223,50 +223,71 @@ file at a specific position, or resize the file.
   calling `close()` or when the stream is automatically closed by the pipe.
 {% endAside %}
 
-### Storing file handles in IndexedDB
+### Storing file handles or directory handles in IndexedDB
 
-File handles are serializable, which means that you can save a file handle to IndexedDB, or call
-`postMessage()` to send them between the same top-level origin.
+File handles and directory handles are serializable, which means that you can save a file or
+directory handle to IndexedDB, or call `postMessage()` to send them between the same top-level origin.
 
-Saving file handles to IndexedDB means that you can store state, or remember which files a user was
+Saving file or directory handles to IndexedDB means that you can store state, or remember which files
```

Comment:
```suggestion
Saving files or directory handles to IndexedDB means that you can store state, or remember which files
```

Reply:
Not in this case. It's to be read as "file handles or directory handles".